services/database.js

```diff
@@ -77,10 +81,24 @@ function Database() {
       };
     }
 
+    let { dbConnectionUrl, dbHostname } = options;
+    if (databaseDialect === 'mssql') {
+      if (hasInstanceName(dbConnectionUrl)) {
+        const [prefix, suffix] = dbConnectionUrl.split('\\');
+        const indexOfColon = suffix.indexOf(':');
```

Comment:
What if there is not port in the connection URL (could happen I think)?
shouldn't we look for the '/' separator after the port?